packages/tezos-tooling/tezos-tooling.7.0~rc1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "1.11" }
+  "ocamlformat" { = "0.10" }
```

Comment:
Do you need to propagate this ocamlformat constraint to all users of the package? It may be better to add `{dev}` here so that only pinned packages (which can then be edited) get the ocamlformat version constraint.

Reply:
`tezos-tooling` is only a `{ with-test }` dependency of other packages so "normal users" should never have it installed.
On the contrary, checking that the code follows ocamlformat rules is part of the tests and ocamlformat rules changes with version so yes, sadly, the tests fails as soon as you relax this constraint.
I'm not the expert here so if there's a better solution, I'll implement it

Reply:
Making it a `with-test` dependency is just fine.  The majority of opam installations (aside from CI) will not use that.